### PR TITLE
chore: started adding dom tags + readme updates to ui elements

### PR DESCRIPTION
**Pull request by JonathanConn**
# Changes

- chore: started adding dom tags + readme updates to ui elements

Was going back and forth a lot building tests, adding tags
Decided to just to overshoot and add most of the remaining tags

Need to finish adding remaining views

### CODE REVIEW

File: packages/ui/readme.md [hunk 1]
```diff
@@ -17,3 +17,111 @@ Your on-ramp to web3 multichain. Web3Modal is a versatile library that makes it
 ## Development
 
 Please follow [developer docs](./.github/docs/development.md) to set up web3modal locally.
+
```
Comment (0xAsimetriq) [inline packages/ui/readme.md:20]: We need to put these docs under `.github/docs` somewhere as package readme's are user facing / displayed on npm.

Comment (JonathanConn) [reply packages/ui/readme.md:20]: Good idea, I will move it

**Review by 0xAsimetriq — COMMENTED** [review]
Another bit (which you might've planned for latter), we probably want one index file with all of these as constants, then we apply constants throughout the code.

Having such file will mean we only need to tweak names in one place and tests can import them from there as well.

**Comment by JonathanConn** [comment]
@0xAsimetriq You mentioned we moved to cloudflare pages, is it safe to ignore the failed vercel deployments?

**Review by 0xAsimetriq — APPROVED** [review]
Looked over this again and got following question: Do we need testids on "components" i.e. each button within the modal will have these added now, and they accumulate quiet a lot to increase bundle size slightly.

Do we not want these only on concrete actionable elements instead? For example instead of having `data-testid=button-component` show up on all buttons, we probably want `data-testid=connect-btn` / `data-testid=view-all-btn` etc?

If you do need them on all components, we can keep as is 👍

**Pull request merged**